backend/apps/owasp/api/search/project.py

```
@@ -1,8 +1,8 @@
 """OWASP app project search API."""
 
-from algoliasearch_django import raw_search
+from apps.common.typesense import Typesense
+from apps.core.utils.params_mapping_typesense import get_typesense_params_for_index
```

Ruff (0.8.2)

`apps.core.utils.params_mapping_typesense.get_typesense_params_for_index` imported but unused

Remove unused import: `apps.core.utils.params_mapping_typesense.get_typesense_params_for_index`

(F401)